src/domain/challenge/common/JourneyFilter/JourneyFilter.tsx

```diff
@@ -19,15 +27,26 @@ const JourneyFilter = <T extends Identifiable>({
   title,
   inputFieldEnabled = true,
   valueGetter,
+  tagsGetter,
   children,
 }: CardFilterProps<T>) => {
   const [terms, setTerms] = useState<string[]>([]);
 
+  const allValues = getAllValues(data, tagsGetter)
+    .sort((a, b) =>
+      a.count !== b.count
```

Comment:
I was doing it this way, but I hesitated about the stability of Javascript `sort` function. I've read that is only stable since ECMAScript2019, was dependent on the browser, on the number of items in the array....
But yeah, after seeing your comment I've investigated a bit more, and looks like almost all modern browsers have a stable sort function: https://stackoverflow.com/a/3027715 
This code is cleaner